libs/ui-watt/project.json

```diff
@@ -23,15 +22,13 @@
       "outputs": ["{options.outputFile}"]
     },
     "storybook": {
-      "executor": "@nrwl/storybook:storybook",
+      "executor": "@storybook/angular:start-storybook",
       "options": {
         "port": 4400,
         "host": "0.0.0.0",
-        "projectBuildConfig": "ui-watt:build-storybook",
-        "uiFramework": "@storybook/angular",
-        "config": {
-          "configFolder": "libs/ui-watt/.storybook"
-        }
+        "configDir": "libs/ui-watt/.storybook",
+        "browserTarget": "ui-watt:build-storybook",
+        "compodoc": false
```

Comment:
Could this option actually let us get rid of our custom compodoc script (`docs:json`) setting this to `true`?

Reply:
Good question. Haven't tried it. Will do and write back whether it's possible. It'd be nice it if is :)

Reply:
It worked. A couple of arguments had to be moved to this file, though. It's pushed now.